internal/executor/disabled_error_cooldown_test.go

```
@@ -113,6 +140,59 @@ func TestDispatchRetriesCommittedStreamReadErrorWhenErrorCooldownEnabled(t *test
 	}
 }
 
+func TestDispatchRetriesInterruptedResponseStreamError(t *testing.T) {
+	proxyRepo := &recordingProxyRequestRepo{}
+	attemptRepo := &recordingAttemptRepo{}
+	calls := 0
+	rec := httptest.NewRecorder()
+	req := httptest.NewRequest(http.MethodPost, "/v1/chat/completions", nil).WithContext(context.Background())
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)